modules/coinswap/client/rest/tx.go

```diff
@@ -188,13 +188,10 @@ func swapOrderHandlerFn(cliCtx client.Context, isBuyOrder bool) http.HandlerFunc
 			return
 		}
 
-		var recipientAddress sdk.AccAddress
-		if len(req.Output.Address) > 0 {
-			recipientAddress, err = sdk.AccAddressFromBech32(req.Output.Address)
-			if err != nil {
-				rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
-				return
-			}
+		recipientAddress, err := sdk.AccAddressFromBech32(req.Output.Address)
+		if err != nil {
+			rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
+			return
```

Comment:
Add check
```go
if senderAddress.Empty() {
}
```
and 
```go
if recipientAddress.Empty() {
}
```

Reply:
No need, because the `AccAddressFromBech32` function has been verified